backends/ebpf/ebpfutils.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# Copyright 2013-present Barefoot Networks, Inc.
+# Copyright 2018 VMware, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Defines helper functions which are used across all eBPF Python files."""
```

Comment:
ebpf python files is not very clear; I would say that this is a library of useful functions used by multiple python scripts

Reply:
You may want to migrate this file into the tools folder and we can reuse this code in the bmv2 back-end.